learn/index.html

```diff
@@ -154,6 +154,14 @@
 
       <ul class="cards">
         <li>
+          <h3>Lunch &amp; Learn</h3>
```

Comment:
I think I actually would rather have the talk titles in the `h3` elements and just omit the event title, which would otherwise be below it.